### PR TITLE
Readme deps

### PR DESCRIPTION
**Pull request by enxebre**
Update dependencies docs

### CODE REVIEW

**Comment by enxebre** [comment]
ok to test

File: README.md [hunk 1]
```diff
@@ -75,6 +75,34 @@ These instructions can be used for AWS:
...
+```
+rm glide.lock
+glide install --strip-vendor
+glide-vc --use-lock-file --no-tests --only-code
+bazel run //:gazelle
+```
+
+If it worked correctly it should:
+- Clone your new dep to the `/vendor` dir, and check out the ref you specified.
+- Update `glide.lock` to include your new package, adds any transitive dependencies, and updates its hash.
```
Comment (squat) [inline README.md]: `adds` should be `add` and `updates` should be `update`

File: README.md [hunk 2]
```diff
@@ -75,6 +75,34 @@ These instructions can be used for AWS:
...
+
+```
+rm glide.lock
+glide install --strip-vendor
+glide-vc --use-lock-file --no-tests --only-code
+bazel run //:gazelle
+```
+
+If it worked correctly it should:
+- Clone your new dep to the `/vendor` dir, and check out the ref you specified.
```
Comment (squat) [inline README.md]: no comma needed here

File: README.md [hunk 3]
```diff
@@ -75,6 +75,34 @@ These instructions can be used for AWS:
...
+- Update `glide.lock` to include your new package, adds any transitive dependencies, and updates its hash.
+- Regenerate BUILD.bazel files.
+
+For the sake of your fellow reviewers, commit vendored code changes as a separate commit from any other changes.
```
Comment (squat) [inline README.md]: it's a little bit funny to say `commit x as a commit`. maybe `commit vendored code changes separately from any other changes`

File: README.md [hunk 4]
```diff
@@ -75,6 +75,34 @@ These instructions can be used for AWS:
     tectonic destroy --dir=$CLUSTER_NAME
     ```
 
+### Managing Dependencies
+#### Go
+
+We follow a hard flattening approach, direct and inherited dependencies are installed in the base `vendor/`.
```
Comment (squat) [inline README.md]: a comma is odd here. use `;` or maybe `i.e.`

File: README.md [hunk 5]
```diff
@@ -75,6 +75,34 @@ These instructions can be used for AWS:
...
+
+Dependencies are managed with [glide](https://glide.sh/), but committed directly to the repository. If you don't have glide, install the latest release from [https://glide.sh/](https://glide.sh/). We require version 0.12 at a minimum.
```
Comment (squat) [inline README.md]: no comma needed before `but`

**Comment by enxebre** [comment]
retest this please

**Comment by enxebre** [comment]
retest this please

**Pull request merged**